### ISSUE TITLE
Error with web plugin Neighbors.pm

### ISSUE THREAD

**Issue by ollyg** [netdisco/netdisco#527]
```
~> ~/bin/localenv starman /usr/local/netdisco/bin/netdisco-web-fg
--workers=1 --disable-keepalive

2019/03/04-12:14:02 Starman::Server (type Net::Server::PreFork)
starting! pid(6063)
Resolved [*]:5000 to [::]:5000, IPv6
Not including resolved host [0.0.0.0] IPv4 because it will be handled
by [::] IPv6
Binding to TCP port 5000 on host :: with IPv6
Setting gid to "100 100 16 33 100"
[6064] 2019-03-04 11:14:02  warn App::Netdisco 2.040005 web
Array found where operator expected at
/usr/local/netdisco/perl5/lib/perl5/App/Netdisco/Web/Plugin/Device/Neighbors.pm
line 129, at end of line
        (Missing operator before ?)
Error while loading /usr/local/netdisco/bin/netdisco-web-fg: syntax
error at /usr/local/netdisco/perl5/lib/perl5/App/Netdisco/Web/Plugin/Device/Neighbors.pm
line 129, near "List::MoreUtils::zip6
    @"
syntax error at
/usr/local/netdisco/perl5/lib/perl5/App/Netdisco/Web/Plugin/Device/Neighbors.pm
line 129, near "}}"
Global symbol "$link" requires explicit package name at
/usr/local/netdisco/perl5/lib/perl5/App/Netdisco/Web/Plugin/Device/Neighbors.pm
line 129.
Global symbol "$link" requires explicit package name at
/usr/local/netdisco/perl5/lib/perl5/App/Netdisco/Web/Plugin/Device/Neighbors.pm
line 130.
Global symbol "$link" requires explicit package name at
/usr/local/netdisco/perl5/lib/perl5/App/Netdisco/Web/Plugin/Device/Neighbors.pm
line 130.
Global symbol "$left_name" requires explicit package name at
/usr/local/netdisco/perl5/lib/perl5/App/Netdisco/Web/Plugin/Device/Neighbors.pm
line 133.
Global symbol "$right_name" requires explicit package name at
/usr/local/netdisco/perl5/lib/perl5/App/Netdisco/Web/Plugin/Device/Neighbors.pm
line 134.
Global symbol "@zipped" requires explicit package name at
/usr/local/netdisco/perl5/lib/perl5/App/Netdisco/Web/Plugin/Device/Neighbors.pm
line 134.
syntax error at
/usr/local/netdisco/perl5/lib/perl5/App/Netdisco/Web/Plugin/Device/Neighbors.pm
line 135, near "}"
syntax error at
/usr/local/netdisco/perl5/lib/perl5/App/Netdisco/Web/Plugin/Device/Neighbors.pm
line 274, near "}"
/usr/local/netdisco/perl5/lib/perl5/App/Netdisco/Web/Plugin/Device/Neighbors.pm
has too many errors.
Compilation failed in require at
/usr/local/netdisco/perl5/lib/perl5/Module/Load.pm line 77.
Can't locate App/Netdisco/Web/Plugin/Device/Neighbors in @INC (@INC
contains: /usr/local/netdisco/perl5/lib/perl5/auto/share/dist/App-Netdisco/lib
/usr/local/netdisco/perl5/lib /usr/local/netdisco/perl5/bin/lib
/usr/local/netdisco/perl5/lib/perl5/5.10.0/x86_64-linux-thread-multi
/usr/local/netdisco/perl5/lib/perl5/5.10.0
/usr/local/netdisco/perl5/lib/perl5/x86_64-linux-thread-multi
/usr/local/netdisco/perl5/lib/perl5
/usr/lib/perl5/5.10.0/x86_64-linux-thread-multi /usr/lib/perl5/5.10.0
/usr/lib/perl5/site_perl/5.10.0/x86_64-linux-thread-multi
/usr/lib/perl5/site_perl/5.10.0
/usr/lib/perl5/vendor_perl/5.10.0/x86_64-linux-thread-multi
/usr/lib/perl5/vendor_perl/5.10.0 /usr/lib/perl5/vendor_perl .) at
/usr/local/netdisco/perl5/lib/perl5/Module/Load.pm line 77.
Compilation failed in require at
/usr/local/netdisco/bin/netdisco-web-fg line 53.
BEGIN failed--compilation aborted at
/usr/local/netdisco/bin/netdisco-web-fg line 53.
[6065] 2019-03-04 11:14:02  warn App::Netdisco 2.040005 web
```

**Comment by def-bs** [comment]
I have got the same issue?

Is there a way to fix it?

**Comment by ollyg** [comment]
Issue is outdated version of List::MoreUtils.
Fixed in 2.040006 which is going to CPAN now.
